NightscoutUploadKit/DeviceStatus/PredictedBG.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PredictedBG.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 8/8/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct PredictedBG {
+    let values: [Double]
+    let cob: [Double]?
+    let iob: [Double]?
+
+    public init(values: [Double], cob: [Double]? = nil, iob: [Double]? = nil) {
```

Comment:
NS doesn't *need* them; they're optional.  It's used to display a glucodyn like graph, where the carb effect line is shown on top, insulin on bottom, and combined (actual bg): https://files.gitter.im/nightscout/public/als3/Screenshot-2016-06-03-23.37.04.png